roadmap.md

```diff
@@ -0,0 +1,8 @@
+# roadmap
+## tool to generate a working vice config script complete w lauch
+- determine working vice command line options (al's script?)
+- look up each option, create a dialog that has all required options
```

Comment:
I would suggest using multiple dialogs. I'll outline my thoughts on the project.

Reply:
I'm assuming we're using bash here?

[whiptail](https://manpage.me/index.cgi?apropos=0&q=whiptail&sektion=0&manpath=Debian+8.1.0&arch=default&format=html) is a text-mode dialog generator
[qdialog](https://doc-snapshots.qt.io/qtforpython-dev/PySide2/QtWidgets/QDialog.html) is more Python-oriented (and therefore GUI)

Things this script should do (not sure how dialogs should be broken up yet):
- [ ] Determine whether the host OS has `whiptail` or `qdialog` available
- [ ] Determine whether VICE ROMs are present (by default, kernal and 15x1 drive ROMs are not supplied with Debian VICE package because they're non-free)
- [ ] Offer to download them off zimmers.net (wget?)
- [ ] Al has also provided ROMs with "Image 2.0 on a stick", don't know if it's feasible to use them
- [ ] Copy to `/usr/lib/vice/ROMs` or whatever the proper place is for host OS
- [ ] Determine sysop storage requirements: possible options could include 4 1541s, 4 1581s, Lt. Kernal HD emulation, Creative Micro Designs HD emulation
- [ ] if CMD emulation, provide link to Go4Retro store to [purchase CMD HD boot ROM](http://store.go4retro.com/commodore/cmd-hdd-boot-rom-2-80-binary-image/), verify boot ROM is in place
- [ ] if LtK emulation, either a) warn user the emulated C64 will hang for a while during searching for sysgen'd LtK, or b) use pre-sysgen'd disks (Todd from FB has a start on it)
... some other stuff maybe? ...
- [ ] determine RS232 interface: ACIA or user port (separate VICE command line switches for either option)

All this to ultimately generate a script file to automate launching Image BBS. Should be an interesting experiment.

![image](https://user-images.githubusercontent.com/7505860/102553456-b9f41880-4077-11eb-8716-89fce2f11f82.png)
![image](https://user-images.githubusercontent.com/7505860/102553497-caa48e80-4077-11eb-965a-a6b4c76f51ab.png)

Reply:
Agh, like @jonez734 pointed out, there's compiling the nightly version of VICE, too.

Reply:
I think as a start we should "assume" a properly built and feature-enabled vice and tcpser. focus on being able to generate the right command line options, put them into a bash script, and done. we should track vice and tcpser *very* carefully re: dtr hangup.

Reply:
That would make things a _lot_ easier.